db/migrate/20211202054844_create_keywords.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateKeywords < ActiveRecord::Migration[6.1]
+  def change
+    create_table :keywords do |t|
+      t.string :keyword
+      t.integer :status, default: 0
```

Comment:
I think we can null constraint for this column too

Reply:
Nice catch! Fixed in [`f42675d` (#40)](https://github.com/suho/google-search-ruby/pull/40/commits/f42675dd1e9b7fdfe0b06e96e5e8709d750dd695)